datalad/support/sshconnector.py

```diff
@@ -96,9 +101,11 @@ def open(self):
         connection, if it is not there already.
         """
 
+        # set control options
+        ctrl_options = ("-o ControlMaster=auto %s -o ControlPersist=yes" % ' '.join(self.ctrl_options))
```

Comment:
["-o", "ControlMaster=auto", ...

On July 12, 2016 8:57:37 PM EDT, Debanjum <notifications@github.com> wrote:
>> @@ -96,9 +101,11 @@ def open(self):
>>          connection, if it is not there already.
>>          """
>>  
>> +        # set control options
>> +        ctrl_options = ("-o ControlMaster=auto %s -o
>ControlPersist=yes" % ' '.join(self.ctrl_options))
>
>You mean something like `ctrl_options = ["-o ControlMaster=auto", "-o
>ControlPersist=yes"] + self.ctrl_options` ? this is just being used by
>open command...
>
>---
>You are receiving this because you commented.
>Reply to this email directly or view it on GitHub:
>https://github.com/datalad/datalad/pull/633/files/2c890d4f66eefc95dab0d2b5629c0e212374334a#r70549643